nix/modules/tezos-baker.nix

```diff
@@ -27,6 +37,12 @@ let
         '';
       };
 
+      liquidityBakingToggleVote = mkOption {
+        type = types.enum [ "on" "of" "pass" ];
+        default = "pass";
+        description = "Vote to continue (option "on") or end (option "off") the liquidity baking subsidy. Or choose to pass (option "pass")";
```

Comment:
Does nix handle the quotes-inside-quotes correctly? I'd use multi-line string for description as in the previous option, for consistency and to keep everything ~80 chars wide, but at the minimum I suspect you'd have to escape them.